eq-author-api/schema/resolvers/base.js

```diff
@@ -221,6 +222,7 @@ const Resolvers = {
         ...args.input,
         createdBy: ctx.user.id,
       });
+      await createComments(questionnaire.id);
```

Comment:
Gonna need to add a migration to retroactively add comments objects to previous questionnaires.

Reply:
ahh, good point! - can this be a seperate ticket or must go in with this PR?

Reply:
done